src/core/public/chrome/chrome_service.tsx

```diff
@@ -496,3 +534,19 @@ export interface InternalChromeStart extends ChromeStart {
    */
   getHeaderComponent(): JSX.Element;
 }
+
+/** @public */
+export interface ChromeUseCase {
+  id: string;
+  title: string;
+  description: string;
+  order?: number;
+  icon?: string;
+  reserved?: boolean; // use cases like data administration / settings and setup is not configurable through workspace
+}
+
+/** @public */
+export interface ChromeRegistrationNavLink
```

Comment:
It seems we only need `id` and `category` for now, shall we minimize this interface for the initial implementation?

Reply:
Sure, let me make this interface simple for initial implementation